requirements-dev.txt

```diff
@@ -3,3 +3,4 @@ ipdb
 jupyter
 flake8
 isort
+autopep8
```

Comment:
For this change, I would have preferred it be bundled in a patch other than "cleanup" (maybe even in its own "Add missing linting requirement" commit). That being said, building a meticulous commit history is probably less important than quickly iterating on research code like you are doing right now.

Reply:
Ah, this isn't actually a linting requirement